forte/data/data_store.py

```diff
@@ -247,6 +247,59 @@ def _new_group(self, type_name: str, member_type: str) -> List:
         entry += len(self._type_attributes[type_name]) * [None]
         return entry
 
+    def _is_subclass(
+        self, type_name: str, cls, include_onto_file: bool = False
+    ) -> bool:
+        r"""This function determines if a string class_type is
+        the subclass of a class.
+
+        Args:
+            type_name: A fully qualified name of an entry class
+            cls: A class.
+            include_onto_file: A boolean value whether the class is in user provided ontology file
```

Comment:
I don't think we need to use `include_onto_file` value here. This function should check `_type_attributes` and python environment. I don't understand what this boolean is doing here.